wikipendium/wiki/views.py

```diff
@@ -107,7 +109,15 @@ def add_tag_to_article(request, slug):
     if 'tag' not in request.POST:
         return HttpResponseBadRequest()
 
-    tag = slugify(request.POST['tag'])
+    slug = original_slug = slugify(request.POST['tag'])
```

Comment:
Right! What if there's no tag? should we perhaps do a `original_tag = request.POST.get('tag', None)` or something?